File: IT1110_BaiTapC/IT1110_Codeforces/Bai_4_va_5_cach_2/Bai_4.2/Bai_4.2.c
#include<stdio.h>
#include<math.h>

int main()
{
    int n, i;
    scanf("%d", &n);

    if (n < 2) printf("Khong phai la so nguyen to!");
    else
    {
        for (i = 2; i < sqrt(n); i++)
        {
            if (n % i == 0) 
            {
                printf("Khong la so nguyen to!");
                break;
            }
        }
        printf("La so nguyen to!");
        
    }
    
}

File: Python-learn-IT3150-ProjectI/thuc-hanh-TinDC-byPython/BaiTHso4/bai4.2.py
# Bài 4.2 Số nguyên tố là một số nguyên dương chỉ chia hết cho 1 và chính nó. Viết
# chương trình nhập một số nguyên từ bàn phím, kiểm tra xem số nguyên đó có
# phải là số nguyên tố hay không và thông báo ra màn hình.

import math

def check_prime(n):
    if n <= 1: return False
    elif n == 2: return True
    else:
        flag = True
        for i in range(2, round(math.sqrt(n))+1):
            if n % i == 0:
                flag = False
                break
        return flag
def main():
    n = int(input('Nhap vao so nguyen n: '))
    if check_prime(n):
        print(f'{n} la so nguyen to')
    else:
        print(f'{n} khong la so nguyen to')
        
main()